BOOST-786: update new docs (#228)
boostercloud/booster: docs/README.md

    readonly cartItems: Array<CartItem>,
    public paid: boolean
  ) {}

  @Projection(Cart, 'id')
  public static updateWithCart(cart: Cart, oldCartReadModel?: CartReadModel): CartReadModel {
    return new CartReadModel(cart.id, cart.cartItems, cart.paid)
  }
}
```

#### Read models naming convention

#### Creating a read model
```shell
boost new:read-model CartReadModel --fields id:UUID cartItems:"Array<CartItem>" paid:boolean --projects Cart
```

This will create a file in the read-models directory `project-root/src/read-models/CartReadModel.ts`.

Read Model classes can also be created by hand and there are no restrictions regarding the place you put the files. The structure of the data is totally open and can be as complex as you can manage in your projection functions.

#### The projection function
A `Projection` is a method decorated with the `@Projection` decorator that, given a new entity value and (optionally) a previous read model state, generate a new read model value.

Read models can be projected from multiple [entities](#4-entities-and-reducers) as soon as they share some common key called `joinKey`.

#### Authorizing read models

#### Querying a read model

> Defining a read models enables a new REST Read endpoint that you can use to query or poll the read model records [see the API documentation](#GraphQL-API).

Reviewer: ```suggestion
You can use the GraphQL endpoint to query or subscribe to the read model records: [see the API documentation](#GraphQL-API).
```

Author: Committing the suggestion directly, thanks!
